x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_chart_wrapper.tsx

```diff
@@ -66,6 +67,8 @@ export const WaterfallChartWrapper: React.FC<Props> = ({ data }) => {
 
   return (
     <WaterfallProvider
+      total={total}
```

Comment:
Nit: I think the naming of total and events count can get confusing. Maybe `totalRequests` `totalEvents`. or `requestsCount` and `eventsCount`? I'd prefer total to be a bit more descriptive, and I also think it would be good if the wording used matched somewhat. Not a huge deal at all. Feel free to ignore.